modin/pandas/io.py

```diff
@@ -244,7 +244,18 @@ def read_json(
     compression="infer",
     nrows: Optional[int] = None,
     storage_options: StorageOptions = None,
-):
+):  # noqa: PR01, RT01
+    """
+    Convert a JSON string to Modin object.
+
+    Parameters notes:
+
+    +-----------------+-----------------+----------------+----------------+----------------------------------+
+    | Parameters      | PandasOnRay     | PandasOnDask   | OmniSci        | Notes                            |
+    +=================+=================+================+================+==================================+
+    | All parameters  | Partial         | Partial        | Partial        | Implemented for``lines=True``    |
```

Comment:
```suggestion
    | All parameters  | Partial         | Partial        | Partial        | Implemented for ``lines=True``    |
```
We should probably add a note  in docs what `Notes` stands for and which it is applicable to.